lib/admin_api.ts

```diff
@@ -66,7 +66,9 @@ const azureFunctionHandler = createAzureFunctionHandler(app);
 
 // Binds the express app to an Azure Function handler
 export function index(context: IContext<{}>): void {
-  configureAzureContextTransport(context, winston, "debug");
+  const isProduction = process.env.NODE_ENV === "production";
```

Comment:
I would put this line at the module level, just near the other initialization from env vars